### PR TITLE
chore: filter text inscriptions

### PR DESCRIPTION
**Pull request by topether21**
Hide text/json inscriptions by default on marketplace but not in "Your collection".
It fetches and shows no text inscriptions with priority.

image

image

image

### CODE REVIEW

**Comment by andredevjs** [comment]
On sale area is acting up

https://recordit.co/10owRmWPNj

**Review by andredevjs — CHANGES_REQUESTED** [review]

File: src/utils/openOrdex.js [hunk 1]
```diff
@@ -91,7 +91,11 @@ export async function getOrderInformation(order) {
     // TODO: Remove this call, not needed.
     const inscription = await getInscriptionDataById(inscriptionId);
 
-    validatePbst(sellerSignedPsbt, inscription.output);
+    try {
+        validatePbst(sellerSignedPsbt, inscription.output);
```
[the remark this answers is not in the document]
Comment (topether21) [reply src/utils/openOrdex.js:95]: Oki, it was stopping the flow, let me revert it

**Comment by topether21** [comment]
> On sale area is acting up
> 
> https://recordit.co/10owRmWPNj

fixed

File: src/services/nostr-relay.js [hunk 2]
```diff
@@ -23,25 +28,25 @@ class NostrRelay {
...
                 this.subscriptionOrders = this.subscribe(
                     [{ kinds: [NOSTR_KIND_INSCRIPTION], limit }],
                     async (event) => {
-                        console.log("event", event);
                         try {
                             const order = await getOrderInformation(event);
-
                             if (order) observer.next(order);
+                            i += 1;
+                            console.log("loading", i); // Please keep (little bit) this log for debugging
```
Comment (andredevjs) [inline src/services/nostr-relay.js:43]: you can use console.count instead

**Pull request merged**